VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -444,7 +444,22 @@ def get_total_used_size(self):
             //Centos72test/cifs_test                                cifs      52155392 4884620 47270772  10% /mnt/cifs_test2
 
             '''
-            output = self.command_output_from_subprocess(["df" , "-k" , "--output=source,fstype,size,used,avail,pcent,target"], 30)
+            onlyLocalDisks = False
+            try:
+                configfile='/etc/azure/vmbackup.conf'
```

Comment:
Check if we can reuse existing util method for fetching these similar to happening at below pointer:
https://github.com/Azure/azure-linux-extensions/blob/a81f31479ceaf53742eaeb9fe16af8c9a29b3203/VMBackup/main/freezesnapshotter.py#L84

Reply:
Using existing method